ckanext/datajson/harvester_datajson.py

```diff
@@ -26,7 +26,7 @@ def info(self):
     def load_remote_catalog(self, harvest_job):
         url = harvest_job.source.url
         # todo: into config and across harvester
```

Comment:
any idea what this todo is for?

Reply:
Yeah, ideally all catalog harvester request should use the custom User Agent. Until now we only implemented it for data.json type of harvester. The TODO remains to be done.